src/config/types.ts

```diff
@@ -1,19 +1,40 @@
-import { UnderlyingCollateralToken } from '../types';
+import { Provider } from 'ethers';
+import { Decimal } from '@tempusfinance/decimal';
+import { Token, UnderlyingCollateralToken } from '../types';
 
 export type SupportedNetwork = 'goerli' | 'mainnet';
 
-type TickerToAddressMap = {
+type UnderlyingTokenTickerToCollateralAddressMap = {
   [tokenTicker in UnderlyingCollateralToken]: string;
 };
```

Comment:
As I said in an earlier comment, maybe it would be better to use a `Record` here to force us to check if we forget to add new underlying collateral.

```suggestion
type UnderlyingTokenTickerToCollateralAddressMap = Record<UnderlyingCollateralToken, string>;
```

In this case, we won't need any of those types, as we can use the `Record`